CRM/Core/DAO/PrevNextCache.php

```diff
@@ -173,6 +180,19 @@ public static function &fields() {
           'bao' => 'CRM_Core_BAO_PrevNextCache',
           'localizable' => 0,
         ],
+        'expiry_date' => [
+          'name' => 'expiry_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Prev Next Expiry Date'),
+          'description' => ts('The Time this prev_next_cache data set expires'),
+          'required' => TRUE,
+          'where' => 'civicrm_prevnext_cache.expiry_date',
+          'default' => 'current_timestamp()',
```

Comment:
I think default should be NULL

Reply:
which then means its a not required field i guess?